### PR TITLE
Added Migration File for Providing Initial `PronounSeries`and `Gender` Data

### PR DESCRIPTION
**Pull request by farooqashar**
This PR adds a new migration file named `0006_initial_data.py` to the existing set of migrations file. The purpose of this file, when the project is migrated to the latest version, is to provide initial data to the `PronounSeries` and `Gender` Models/Tables. This is done via a custom function `add_objects` that creates and saves the relevant `PronounSeries` and `Gender` objects. 

As derived from the old set of common pronoun series and genders that were used in the standard toolkit, this PR adds the following objects as initial objects for the database: 

**PronounSeries**

* `he_series`
* `she_series`
* `they_series`
* `xe_series`
* `ae_series`
* `fae_series`
* `ey_series`
* `ve_series`
* `per_series`
* `ze_hir_series`

**Genders**

* `Male` (utilizing the `he_series` pronoun series)
* `Female` (utilizing the `she_series` pronoun series)
* `Nonbinary` (utilizing the `they_series` pronoun series) 
* `Neogenders` (utilizing the `xe_series`, `ae_series`, `fae_series`, `ey_series`, `ve_series`, `per_series`,and `ze_hir_series` pronoun series)

### CODE REVIEW

File: backend/app/migrations/0006_initial_data.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    fae_series = pronounseries.objects.create(identifier = "Fae",subj = "fae",obj = "faer",pos_det = "faer",pos_pro = "faers",reflex = "faerself")
+    ey_series = pronounseries.objects.create(identifier = "Ey",subj = "ey",obj = "em",pos_det = "eir",pos_pro = "eirs",reflex = "eirself")
+    ve_series = pronounseries.objects.create(identifier = "Ve",subj = "ve",obj = "ver",pos_det = "vis",pos_pro = "vis",reflex = "verself")
+    per_series = pronounseries.objects.create(identifier = "Per",subj = "per",obj = "per",pos_det = "per",pos_pro = "pers",reflex = "perself")
+    ze_hir_series = pronounseries.objects.create(identifier = "Ze",subj = "ze",obj = "hir",pos_det = "hir",pos_pro = "hirs",reflex = "hirself")
+
+    all_series = [xe_series,ae_series,fae_series,ey_series,ve_series,per_series,ze_hir_series]
+    Neogenders = gender.objects.create(label = "Neo")
+    for each_series in all_series:
+        Neogenders.pronoun_series.add(each_series)
```
Comment (MBJean) [inline backend/app/migrations/0006_initial_data.py]: This looks great overall! My one comment is that the capital-case variable name is a convention for classes (i.e., `Gender`, `PronounSeries`, etc.), not instances (i.e., `new_pronoun_series = PronounSeries.create(...)`). In lines 7 and 8 above, for instance, to follow the convention would be to write:
```
    PronounSeries = apps.get_model('app', 'PronounSeries')
    Gender = apps.get_model('app', 'Gender')
```

And elsewhere, to use lower case to reference class instances. ex:
```
female = gender.objects.create(label = "Female")
female.pronoun_series.add(she_series)
```

I'd recommend following the convention here, if possible.

**Review by MBJean — CHANGES_REQUESTED** [review]
Great work overall. I have on comment above that I'd like to see addressed, but when that's done this'll be ready to go into production.

File: backend/app/migrations/0006_initial_data.py (hunk 1 above)
Comment (farooqashar) [reply backend/app/migrations/0006_initial_data.py]: This makes sense. I have gone back and followed that convention in the migrations file.

**Review by MBJean — APPROVED** [review]
LGTM. Thanks for addressing my feedback.

**Pull request merged**